src/components/Display.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+function Display(props) {
```

Comment:
**[OPTIONAL]**
* [ ] we can equally destructure at the argument level. It looks cleaner
`function Display({result}) {`